app/views/shf_applications/_unpaid_new_member.html.haml

```diff
@@ -0,0 +1,17 @@
+- if current_user.membership_expire_date.nil?
+  %p Du är godkänd och nu är det dags att betala dina avgifter.
+  %h4 Ditt medlemskap betalas på denna sida:
+  = link_to('Betala medlemsavgift', user_path(current_user), class: 'btn btn-info')
+  %br
+  %br
+- elsif current_user.membership_expire_date.past?
+  %p Du är godkänd sedan innan men du har just nu en eller flera avgifter som inte är betalda.
+  %h4 Ditt medlemskap har slutdatum #{current_user.membership_expire_date}
+  = link_to('Betala medlemsavgift', user_path(current_user), class: 'btn btn-info')
+  %br
+  %br
+- if current_user.has_company? && current_user.company.branding_expire_date.nil?
+  %h4 Din H-märkningspremie behöver betalas:
+  = link_to('Betala H-märkningspremie', company_path(current_user.company), class: 'btn btn-info')
+  %br
+  %br
```

Comment:
There is a great deal of duplication within and across this and the prior file.

See my PR (against your PR) for my refactoring.